### ISSUE TITLE
Editor: Update view post button to open in-app

### ISSUE THREAD

**Issue by jeremeylduvall** [Automattic/wp-calypso#11981]
After #11459 lands, we'll be using the in-app view from both the Posts and Pages lists. Here's a screenshot of how it appears:

<img width="2207" alt="f7ba7a74-f4e0-11e6-90e8-416314c99395" src="https://cloud.githubusercontent.com/assets/7240478/23774661/0455a478-04e2-11e7-8108-dc5df7848110.png">

Per the discussion in #11459, it would be nice to have the same behavior in the success notice that appears in the editor when publishing or scheduling a post. Here's a view of that notice that currently takes you to another tab.

<img width="1007" alt="screen shot 2017-03-09 at 4 05 22 pm" src="https://cloud.githubusercontent.com/assets/7240478/23774712/411cd1c4-04e2-11e7-8a23-f4f15c43652b.png">

**Comment by mtias** [comment]
We should remove the external icon if we do this.
